CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
```

Comment:
On release branch this change to the release [1.6.0] with a date (next Tuesday maybe?)

Reply:
Okay, I will pull the release branch and do that. It slipped out of my mind, sorry.